wt-status.h

```diff
@@ -49,6 +49,11 @@ static inline int is_from_cherry_pick(enum commit_whence whence)
 		whence == FROM_CHERRY_PICK_MULTI;
 }
 
+static inline int starts_with_newline(const char *p)
+{
+	return *p == '\n' || (*p == '\r' && p[1] == '\n'); 
+};
```

Comment:
Since this function is used only in `wt-status.c`, there is no need to define it in a public header. Just insert it before the `wt_status_locate_end()` function.

In any case, the trailing semicolon after the closing curly bracket is superfluous and needs to be removed.